merkle-tree/src/lib.rs

```diff
@@ -1,5 +1,3 @@
-#![no_std]
```

Comment:
Was this just for debugging? Would prefer to keep no_std

Reply:
yes it was, sorry about that